packages/hub/services/scheduled-payments/fetcher.ts

```diff
@@ -0,0 +1,61 @@
+import { inject } from '@cardstack/di';
+import { startOfDay, subDays } from 'date-fns';
+import { convertDateToUTC } from '../../utils/dates';
+
+export default class ScheduledPaymentsFetcherService {
+  prismaManager = inject('prisma-manager', { as: 'prismaManager' });
+  validForDays = 10; // Keep this in sync with the value in the config contract
```

Comment:
Can this be read out of the contract? Or how will it be kept in sync?

Reply:
The idea is to read it from the contract. But I think the contract is not finalized yet - I'll make a ticket to adjust this once we get it deployed.